steps/utils.py

```
@@ -44,28 +53,33 @@ def load_data(train: bool,
     data = []
     directories = next(os.walk(image_dir))[1]
 
     for dirname in directories:
-        logging.info(f'Loading images from the {dirname} directory.')
+        logger.info(f'Loading images from the {dirname} directory.')
         file_names = next(os.walk(os.path.join(image_dir, dirname)))[2]
+        label = label_img(dirname)
 
-        for i in range(configs['n_images_per_dir']):
+        for _ in range(configs['n_images_per_dir']):
             image_name = choice(file_names)
```

Ruff (0.15.20)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)